### ISSUE TITLE
Update screen icons for next release

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/masses-and-springs#374]
**Test device**
Samsung
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/936
The masses on the lab screen are now different colors than they were in published. They were red and blue, and are now pink and cyan. This was likely done to accommodate color blind vision. However, it does mean the mass in the icon for the lab screen does not match any of the actual masses on that screen.

**Visuals**
![labicon](https://user-images.githubusercontent.com/41024075/236549776-2bb5c214-456a-4344-b597-a5db68d81c40.png)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Masses and Springs‬
URL: https://phet-dev.colorado.edu/html/masses-and-springs/1.1.0-dev.2/phet/masses-and-springs_all_phet.html
Version: 1.1.0-dev.2 2023-05-02 23:06:13 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36
Language: en-US
Window: 1536x714
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 4096
Texture: size: 8192 imageUnits: 32 (vertex: 32, combined: 64)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by arouinfar** [comment]
Good catch @KatieWoe. Looks like the screen icons are artwork, so I'll take care of updating it.

**Comment by arouinfar** [comment]
I updated the color of the mass to match the one used in the sim. @jbphet here are the updated assets: 
[updated-lab-screen-icon.zip](https://github.com/phetsims/masses-and-springs/files/11629738/updated-lab-screen-icon.zip)

![labScreenIcon](https://github.com/phetsims/masses-and-springs/assets/8419308/778f16f3-e97a-42b7-850f-d627aea05f96)

**Comment by jbphet** [comment]
Unassigning and marking as "blocks-sim-publication" since the republication of this sim has been deprioritized for now.  This should be addressed (at least to some degree) before republishing.

**Comment by arouinfar** [comment]
PhetColorScheme energy colors were updated in https://github.com/phetsims/scenery-phet/issues/975, so the Energy Screen icon will also need updating.
